components/locid/src/extensions/other/mod.rs

```diff
@@ -19,14 +19,11 @@
 //! let mut loc: Locale = "en-US-a-foo-faa".parse().expect("Parsing failed.");
 //! ```
 
-mod subtag;
-
 use crate::parser::ParserError;
 use crate::parser::SubtagIterator;
 use crate::shortvec::ShortBoxSlice;
+use crate::subtags::Subtag;
```

Comment:
Observation: I would say that you should `pub(use) Subtag`, but that's no longer necessary since 1.5 is out the door